bin/build.mjs

```diff
@@ -55,16 +55,46 @@ async function buildPackage(absWorkingDir) {
    * @param {'esm' | 'cjs'} format
    */
   const entryPoints = (format) => {
-    let outFilename = filename + (format === 'esm' ? '.module' : '');
-    outFilename += DEV ? '' : '.min';
+    /**
+     * @param {string} base
+     */
+    const addSuffix = (base) => {
+      let outFilename = base + (format === 'esm' ? '.module' : '');
+      return outFilename + (DEV ? '' : '.min');
+    };
+    let outFilename = addSuffix(filename);
+
+    const extraEntryPoints = ['@pixiv/three-vrm', '@pixiv/three-vrm-materials-mtoon'].includes(packageJson.name)
+      ? { [addSuffix('nodes/index')]: 'src/nodes/index.ts' }
+      : {};
+
     return {
       entryPoints: {
         [outFilename]: 'src/index.ts',
+        ...extraEntryPoints,
       },
       format,
     };
   };
 
+  /**
```

Comment:
Coundn't find a better way to do this. IMO it's too wasteful to bundle shaders and everything from three-vrm-materials-mtoon main entry again

Reply:
I will find a way to quit `MToonNodeMaterialLoaderPlugin`. Maybe having a factory pattern or something would help this